src/graph/dot.ts

```diff
@@ -80,17 +82,24 @@ function nodeLabel(node) {
     node.name
   }</FONT><BR/>${kindLabel}</TD>
       </TR>
-      ${objectValues(node.fields, nodeField)}
+      ${objectValues(node.fields, nodeField, displayOptions)}
```

Comment:
@soneymathew Can you make the third argument a filter predicate and pass `canDisplayRow`?